networkx/algorithms/centrality/flow_matrix.py

```diff
@@ -71,7 +71,11 @@ def width(self, L):
         m = 0
         for i, row in enumerate(L):
             w = 0
-            x, y = np.nonzero(row)
+            # scipy.sparse indexing for 1D has changed https://github.com/scipy/scipy/pull/20120
+            try:
+                x, y = np.nonzero(row)
+            except ValueError:
+                y = np.nonzero(row)[0]
```

Comment:
The index `[-1]` will work for old and new versions of scipy
```suggestion
            y = np.nonzero(row)[-1]
```